premerge/advisor/advisor_lib_test.py

```diff
@@ -70,6 +70,7 @@ def test_upload_failures(self):
                 {"name": "a.ll", "message": "failed in way 1"},
                 {"name": "b.ll", "message": "failed in way 2"},
             ],
```

Comment:
It looks like you're only testing for Linux failures? Wouldn't it be wise to test Windows failures as well?